contents/shells/bash.md

```diff
@@ -0,0 +1,179 @@
+<frontmatter>
+  title: Introduction to bash shell
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to Linux bash shell 
+
+Author: [Wang Junming](https://github.com/junming403)
+
+Reviewers: [Rahul Rajesh](https://github.com/rrtheonlyone), [Ong Shu Peng](https://github.com/ongspxm), [Jeremy Choo](https://www.github.com/ChooJeremy), [Tan Zhen Yong
+](https://github.com/Xenonym)
+
+## Overview
+
+In general, a computer can be divided into 3 abstraction layers, hardware, kernel and applications. Users cannot control the hardware directly; instructions have to be given through kernel as the kernel is the one that controls the hardware.
+
+<img src="https://cdn-images-1.medium.com/max/1200/1*zEv6mAa2wzHnz4a5uWW4gw.png" alt="shell-kernel" style="width:300px;"/>
+
+<sub>[What (really) happens when you type ls -l in the shell](https://medium.com/meatandmachines/what-really-happens-when-you-type-ls-l-in-the-shell-a8914950fd73)</sub>
+
+The **shell** is the interface through which we pass instructions to the kernel. These instructions will then be executed through the hardware. In this chapter, we will look into **Linux bash shell**.
```

Comment:
Umm... wouldn't it be simpler to say the shell is a way for users to issue commands to a computer? i.e., without mentioning hardware, abstraction, kernal, etc.